### PR TITLE
Remove web forms

### PR DESCRIPTION
**Pull request by CKegel**
Remove web forms on standby, CPR, and Contact Us pages. This is a band aid patch until we get a new website up and running.
Screenshot from 2024-09-14 00-37-07
Screenshot from 2024-09-14 00-37-03
Screenshot from 2024-09-14 00-02-14

### CODE REVIEW

File: views/contact.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <section id="contact-info">
     <div class="center">
         <h2>Contact Us</h2>
-        <p class="lead">Shoot us an email using the form below! Or come visit us!</p>
+        <p class="lead">Shoot us an email at <a href="mailto:officers@rpiambulance.com">officers@rpiambulance.com</a> - Or come visit us!</p>
+        <p class="lead">You can also give us a call at <a href="tel:5189007742">(518) 900-RPIA</a></p>
+        <p class="lead">Expierencing a medical emergency? <span style="color: #c52d2f;"><strong>Dial 911</strong></span></p>
```
Comment (ddbruce) [inline views/contact.html:6]: typo

File: views/contact.html [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <section id="contact-info">
     <div class="center">
         <h2>Contact Us</h2>
-        <p class="lead">Shoot us an email using the form below! Or come visit us!</p>
+        <p class="lead">Shoot us an email at <a href="mailto:officers@rpiambulance.com">officers@rpiambulance.com</a> - Or come visit us!</p>
```
Comment (ddbruce) [inline views/contact.html:4]: please change to:

`&mdash;or come visit us!`

File: views/coverage-details.html
Comment (ddbruce) [inline views/coverage-details.html:1]: Why was this deleted?

File: views/coverage.html [hunk 3]
```diff
@@ -64,97 +50,14 @@ <h1>Tier III</h1>
...
                         <li>5939 (dedicated standby)</li>
                         <li>2+ emergency event supervisors</li>
-                        <li class="plan-action">
-                            <a href="" ng-click="toggleModal('Tier III')" class="btn btn-primary">Select</a>
-                        </li>
                     </ul>
                 </div>
             </div>
         </div>
+        <h3 class="text-center"><br><strong>Please note: A minimum of 2 weeks notice is required for event standbys.</strong></h3>
```
Comment (ddbruce) [inline views/coverage.html:57]: Please change to "two"

**Review by ddbruce — CHANGES_REQUESTED** [review]

File: views/coverage-details.html
Comment (CKegel) [reply views/coverage-details.html:1]: This is a "ghost/dead page" if you will. Note is doesn't reflect the current website and is not referenced anywhere else in the code.

Comment (ddbruce) [reply views/coverage-details.html:1]: 👍

**Review by ddbruce — APPROVED** [review]
Lgtm

**Pull request merged**